templates/subscription-centre/index.html

```diff
@@ -16,48 +25,41 @@ <h1 class="p-heading--2">Subscription centre</h1>
           <span class="p-checkbox__label">Include regular updates about Canonical's products and services, including webinar and event invites</span>
         </label>
         <hr class="p-separator is-shallow">
-        <div class="row">
-          <div class="col-6">
-            <div class="p-accordion">
-              <ul class="p-accordion__list" id="subscriptions-list">
-                {% for category in categories %}
-                  <li class="p-accordion__group">
-                    <div role="heading" aria-level="1" class="p-accordion__heading">
-                      <button type="button" class="p-accordion__tab" id="tab{{ loop.index }}" aria-controls="tab{{ loop.index }}-section" aria-expanded="false"><strong>{{ category }}</strong></button>
-                    </div>
-                    <section class="p-accordion__panel has-tick-elements" id="tab{{ loop.index }}-section" aria-controls="tab{{ loop.index }}-section" aria-hidden="true">
-                      <div class="row">
-                        {% for tag in categories[category] %}
-                          <div class="col-3">
-                            <label class="p-checkbox">
-                              <input type="checkbox" class="p-checkbox__input" name="{{ catagory }}" aria-labelledby="{{ tag }}" value="{{ tag }}" {% if interests is not none and tag in interests %}checked{% endif %}>
-                              <span class="p-checkbox__label" id="{{ tag }}">{{ tag }}</span>
-                            </label>
-                          </div>
-                        {% endfor %}
+        <div class="p-accordion">
+          <ul class="p-accordion__list" id="subscriptions-list">
+            {% for category in categories %}
+              <li class="p-accordion__group">
+                <div role="heading" aria-level="1" class="p-accordion__heading">
```

Comment:
`aria-level="1"` puts them all at the same level as a `h1`:
```suggestion
                <div role="heading" aria-level="2" class="p-accordion__heading">
```